maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
+	desc = "An \"internal use only\" memo from the Chief Science Officer to research staff reminding them of the importance of actually documenting their findings. Some reports are attached highlighting numerous issues with data loss from projects after many scientists showed a tendency to forget their work after cryogenic sleep cycles."
 
 /obj/item/documents/generic/sci/chat
 	name = "chatroom log"
-	desc = "Someone drew a big laughing face over this, but you can still read the contents. 'What do you guys want the most?' 'I wish we had our own security sometimes.'"
+	desc = "Someone drew a big laughing face over this, but you can still read the contents. \"What do you guys want the most?\" \"I wish we had our own security sometimes.\""
 
 /obj/item/document/generic/sci/petrov
 	name = "Petrov map"
-	desc = "It's a portable map of the Petrov. Someone wrote over it in big letters: 'What the fuck is the Petrov?'"
+	desc = "It's a portable map of the Petrov. Someone wrote over it in big letters: \"What the fuck is the Petrov?\", and it's now unusable."
 
 /obj/item/documents/generic/sci/fuckingdoors
 	name = "windoor guide"
-	desc = "\"Internal Use ONLY\" visual guide for the proper operation of xenobiology containment. Features a hand-written note saying "PAY ATTENTION TO THIS!" in large letters."
+	desc = "An \"internal use only\" visual guide for the proper operation of xenobiology containment doors. It features a hand-written note saying \"PAY ATTENTION TO THIS!\" in large letters."
 
 /obj/item/documents/generic/corp/bodyarmourbad
-	name = "LPA recommendations"
-	desc = "\"Internal Use ONLY\" corporate documents, detailing that having any Loss-Prevention Associate wearing bodyarmour and openly carrying a firearm inside of Secure Areas without proper justification actively causes harm to the ability of Corporate to secure permission to allow Corporate Guards onto Government vessels. It also notes that this issue will be brought up at every 3 months during Contract renewal."
+	name = "LPA recommendations"	
+	desc = "Some \"internal use only\" corporate documents, detailing that having loss prevention associates openly wear body armor and a smartgun is an extreme violation of trust between EXO and the SCG and will result in harsh penalties. It also notes that this issue will be brought up at every 3 months during contract renewal."
 
-/obj/item/documents/generic/corp/stipendbudget
+/obj/item/documents/generic/corp/coffeecut
 	name = "corporate spending records"
-	desc = "\"Confidential\" memo on the current usage of funds provided to SEV Torch Corporate Liaisons, apparently there's threats of a strike over a cut to the cost of living stipend by head office."
+	desc = "A \"confidential\" memo on the current usage of funds provided to SEV Torch corporate liaisons; apparently there's threats of a strike over a cut to the coffee budget by the head office."
```

Comment:
I originally had it as "coffee budget" and then Eckles disagreed.
Because we don't charge for coffee.
Cost of Living Stipend has to stay, for vague lore reasons.

Reply:
Ah, looks like it was another merge conflict I didn't catch.